content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,240 @@
+========================================
+Connect Gmail to Odoo using Google OAuth
+========================================
+
+Odoo is compatible with Google's OAuth for Gmail. In order to send secure emails from a custom
+domain, configure a few settings on Google's *Workspace* platform, as well as on the back end of
+the Odoo database. This is necessary to use a Google external email server with both a personal
+(youname@gmail.com) and custom domain (name@yourcompany.com / Google Workspace Account).
+
+.. tip::
+   For more information, visit
+   `Google's documentation <https://support.google.com/cloud/answer/6158849>`__ on setting up
+   OAuth.
+
+Setup in Google
+===============
+
+Create a new project
+--------------------
+
+To get started, go to `Google APIs platform
+<https://console.cloud.google.com/apis/credentials/consent>`__ Log in with your *Google Workspace*
+account if you have one, otherwise log in with your personal Gmail account. Choose the email
```

Comment:
Line 23-24: add parentheses around `Choose the email address you want to configure in Odoo.`